utils/src/random.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use rand::distributions::{Standard, Distribution};
+
+// Random value for `T`.
+pub fn random<T>() -> T
+where
+    Standard: Distribution<T>,
+{
+    rand::random()
+}
+
+// Random Bytestring with random length in range 1..4096.
+pub fn bytestring() -> Vec<u8> {
+    let s = (random::<usize>() % 4095) + 1;
```

Comment:
Better move magic constant outside, or make the function call configurable in terms of range.